simplematrixbotlib/match.py

```diff
@@ -99,18 +121,19 @@ def command(self, command=None):
             Returns the string after the prefix and before the first space if no arg is passed to this method.
         """
 
-        if self._prefix == self.event.body[0:len(self._prefix)]:
-            body_without_prefix = self.event.body[len(self._prefix):]
-        else:
-            body_without_prefix = self.event.body
-
-        if not body_without_prefix:
-            return []
+        if not (self._body_without_prefix and self._body_without_mention):
+            """Body is empty after removing prefix or mention"""
+            if command is None:
+                return ""
+            elif command:
+                return False
+            else:
+                return True
```

Comment:
to improve readability, use the same order of conditions for empty and non-empty bodies